vmprof/reader.py

```diff
@@ -348,6 +348,8 @@ def add_virtual_ip(self, marker, unique_id, name):
 
     def add_trace(self, trace, trace_count, thread_id, mem_in_kb):
         for addr in trace:
+            if not isinstance(addr, NativeCode):
+                continue
```

Comment:
I think this is what is causing tests to fail

Reply:
thanks, indeed